data/scripts.js

```diff
@@ -1715,29 +1715,31 @@ exports.BattleScripts = {
 			item = 'Rocky Helmet';
 		} else if (counter.Physical + counter.Special >= 4 && template.baseStats.def + template.baseStats.spd > 189) {
 			item = 'Assault Vest';
-		} else if (counter.Physical + counter.Special >= 4) {
+		} else if (counter.Physical + counter.Special >= 4 && ability === 'Multiscale') {
 			item = (hasMove['fakeout'] || hasMove['return'] || hasMove['extremespeed'] || (hasMove['suckerpunch'] && !hasType['Dark'])) ? 'Life Orb' : 'Expert Belt';
 		} else if (setupType && hasMove['outrage']) {
```

Comment:
Move the above to here `if (hasMove['outrage'] && (setupType || ability === 'Multiscale'))`